portfolio/src/main/webapp/index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY_HERE"></script>
```

Comment:
Are you inserting your API key here locally, just not committing it to git?

Reply:
Yes, I just replace`YOUR_API_KEY_HERE` with my API key locally.

Reply:
Got it, thanks!